src/middle-end/index.js

```diff
@@ -0,0 +1,33 @@
+const Redux = require('redux');
+const { routerMiddleware: RouterMiddleware } = require('react-router-redux');
+const { middleware: { thunk, errorLogger }, ...M } = require('strange-middle-end');
```

Comment:
I recommend using a different variable here.  Maybe just `MiddleEnd`, in order to distinguish require('strange-middle-end')` from `require('./middle-end')` in the project, which are two different things.